docs/gpu.md

```diff
@@ -41,35 +41,52 @@ In order to match the kOps environment, create a `values.yaml` file with the fol
 operator:
   nodeSelector:
     kops.k8s.io/instancegroup: gpu-nodes
+  tolerations:
+    - key: nvidia.com/gpu
+      operator: Exists
```

Comment:
Please don't indent the sequence values, to keep things consistent. Same in other places.
```suggestion
  - key: nvidia.com/gpu
    operator: Exists
```

Reply:
done

Reply:
Thanks. Please also squash the commits.

Reply:
I'm fed up with the bureaucratic overhead in this repository for a simple change like this. Take it or leave it - no offense.

Reply:
No offense taken, sorry you feel this way. Thank you for your contribution.